src/mir/visitor.md

```diff
@@ -37,11 +37,6 @@ code that will execute whenever a `foo` is found. If you want to
 recursively walk the contents of the `foo`, you then invoke the
 `super_foo` method. (NB. You never want to override `super_foo`.)
 
-A very simple example of a visitor can be found in [`LocalUseVisitor`].
-By implementing `visit_local` method, this visitor counts how many times each local is mutably used.
-
-[`LocalUseVisitor`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_mir_transform/const_debuginfo/struct.LocalUseVisitor.html
-
```

Comment:
is there no other example of a visitor, to replace this one

Reply:
Sure! I've just replaced that with `LocalFinder`.